scripts/borg.win32.exe.spec

```diff
@@ -0,0 +1,51 @@
+# -*- mode: python ; coding: utf-8 -*-
+import os
+
+here = os.path.dirname(os.path.abspath(SPEC))
+basepath = os.path.abspath(os.path.join(here, '..'))
+
+block_cipher = None
+
+a = Analysis([os.path.join(basepath, 'src\\borg\\__main__.py')],
+             pathex=[basepath, ],
+             binaries=[],
+             datas=[
+                 (os.path.join(basepath, 'src', 'borg', 'paperkey.html'), 'borg')
+             ],
+             hiddenimports=[],
+             hookspath=[],
+             runtime_hooks=[],
+             excludes=[
+                 '_ssl', 'ssl',
+             ],
+             win_no_prefer_redirects=False,
+             win_private_assemblies=False,
+             cipher=block_cipher,
+             noarchive=False)
+
+pyz = PYZ(a.pure, a.zipped_data, cipher=block_cipher)
+
+exe = EXE(pyz,
+          a.scripts,
+          a.binaries,
+          a.zipfiles,
+          a.datas,
+          name='borg',
+          debug=False,
+          bootloader_ignore_signals=False,
+          strip=False,
+          upx=True,
+          console=True )
+
+if True: 
+    # Enable this block to build a directory-based binary instead of
+    # a packed single file. This allows one to easily look at all included
+    # files
+    coll = COLLECT(exe,
+                a.binaries,
+                a.zipfiles,
+                a.datas,
+                strip=False,
+                upx=True,
+                upx_exclude=[],
+                name='borg-dir')
```

Comment:
maybe we could have a single borg.exe.spec for linux/freebsd/osx/windows?

for the unavoidable differences, it could also use a `is_win32` switch like in setup.py, but we could at least avoid duplicating all the common code and would only have to do changes at one place.

Reply:
Done, merged the two files and using a platform flag to distinguish between win and linux.